modules/aerospike/aerospike_test.go

```diff
@@ -0,0 +1,57 @@
+package aerospike_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	tcaerospike "github.com/testcontainers/testcontainers-go/modules/aerospike"
+)
+
+const (
+	aerospikeImage = "aerospike/aerospike-server:latest"
+)
+
+// TestAerospike tests the Aerospike container functionality
+// It includes tests for starting the container with a valid image,
+// applying container customizations, and handling context cancellation.
+// It also includes a test for an invalid image to ensure proper error handling.
+// The tests use the testcontainers-go library to manage container lifecycle
+func TestAeroSpike(t *testing.T) {
+	t.Run("invalid-image-fails", func(t *testing.T) {
+		ctx := context.Background()
+		_, err := tcaerospike.Run(ctx, "invalid-aerospike-image")
```

Comment:
bug: while you're expecting an error if it wasn't the container would be orphaned, add testcontainers.CleanupContainer call